File: Modul_1/Soal_10/PRAK110-2310817210015-MuhammadBukhariFitri.py
import math
alas = 5;
tinggi = 12;
sisiMiring = round(math.sqrt((alas * alas) + (tinggi * tinggi)))
keliling = round(alas + tinggi + sisiMiring) 
luas = (alas * tinggi) // 2;

print("Diketahui :");
print(f"Alas = {alas} cm");
print(f"Tinggi = {tinggi} cm\n");
print("Jawab :");
print(f"Sisi A = {tinggi} cm");
print(f"Sisi B = {sisiMiring} cm");
print(f"Sisi C = {alas} cm");
print(f"Keliling = {keliling} cm");
print(f"Luas = {luas} cm");

File: Modul 1/Soal 10/PRAK110-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>
#include <math.h>

int main() {
    int alas = 5;
    int tinggi = 12;
    int sisiMiring = sqrt((alas * alas) + (tinggi * tinggi));
    int keliling = alas + tinggi + sisiMiring; 
    int luas = (alas * tinggi) / 2;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", sisiMiring);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", luas);


    return 0;
}